data/src/format.rs

```diff
@@ -0,0 +1,383 @@
+//! Video frame formats
+
+use std::ops::Index;
+
+use num_traits::FromPrimitive;
+
+use crate::pixel::{
+    ChromaLocation, Chromaton, ColorModel, ColorPrimaries, MatrixCoefficients,
+    TransferCharacteristic,
+};
+
+use crate::plane::PlaneId;
+
+/// Video frame color space representation such as color models, primaries,
+/// color space conversion functions, and chroma location.
+///
+/// This representation also contains additional information about pixels planes
+/// such as their chroma definition.
+#[derive(Clone, Copy, PartialEq, Eq, Debug)]
+pub struct VideoFrameFormat {
```

Comment:
This is a Pixel format, the name is wrong.

Reply:
Just a small explanation on the reasons of this movement. 
 -`pixel.rs` contains color spaces structures and functions
- `format.rs` contains frame pixels representations. My idea would be to add `Soniton`, renamed ad `AudioSampleFormat`, at a second time.

For what concerns the name of the structure, can we use `PixelFrameFormat`?

Reply:
The current names might be a bit unique but they are short and to the point. Renaming them with with something that's longer and/or misleading isn't great.

`AudioFormat` or `format::Audio` and `PixelFormat` or `format::Pixel`, might be short enough, but I would not spend time now on renaming something that's already working.

Reply:
I think I will adopt your `format::name` idea, thanks!